apps/backend/src/mcp/resources/proxy_configs.py

```
@@ -0,0 +1,634 @@
+                            'name': f"{config.service_name} Database Record",
+                            'description': f"Database record with real-time sync for {config.service_name}",
+                            'mime_type': 'application/json'
+                        })
+            
+            except Exception as e:
+                logger.error(f"Error listing configs for device {device_id}: {e}")
+        
+    except Exception as e:
+        logger.error(f"Error listing proxy config resources: {e}")
+    
+    return resources
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)